k4MarlinWrapper/src/components/EDM4hep2Lcio.cpp

```diff
@@ -293,6 +318,9 @@ StatusCode EDM4hep2LcioTool::convertCollections(lcio::LCEventImpl* lcio_event) {
     }
   }
 
+  // TODO: warn here if EventHeader not there
+  info() << "Event: " << lcio_event->getEventNumber() << " Run: " << lcio_event->getRunNumber() << endmsg;
```

Comment:
```suggestion
  info() << "Event: " << lcio_event->getEventNumber() << " Run: " << lcio_event->getRunNumber() << endmsg;
```

better some other verbosity level (debug? verbose?)